openreview/venue/process/invitation_edit_process.py

```diff
@@ -24,21 +24,19 @@ def process(client, invitation):
         print('invitation is not yet active and no child invitations created', cdate)
         return
 
-    def delete_existing_invitations():
-
-        ddate = openreview.tools.datetime_millis(datetime.datetime.now())
-
-        def delete_invitation(child_invitation):
-            client.post_invitation_edit(
-                invitations=meta_invitation_id,
-                readers=[venue_id],
-                writers=[venue_id],
-                signatures=[venue_id],
-                invitation=openreview.api.Invitation(
-                    id=child_invitation.id,
-                    ddate=ddate
-                )
+    def delete_invitation(child_invitation):
+        client.post_invitation_edit(
+            invitations=meta_invitation_id,
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=[venue_id],
+            invitation=openreview.api.Invitation(
+                id=child_invitation.id,
+                ddate=now
             )
+        )
+
+    def delete_existing_invitations():
```

Comment:
I don't think you need to call this function anymore, right?

we are always going to delete the current invitations that were not created in this run

Reply:
Yes, you're right! Forgot to remove it